packages/arb-token-bridge-ui/src/util/deposits/helpers.ts

```diff
@@ -43,11 +44,13 @@ export const updateAdditionalDepositData = async ({
   let timestampCreated = new Date().toISOString()
   if (depositTx.timestampCreated) {
     // if timestamp is already there in Subgraphs, take it from there
-    timestampCreated = String(Number(depositTx.timestampCreated) * 1000)
+    timestampCreated = String(normalizeTimestamp(depositTx.timestampCreated))
   } else if (depositTx.blockNumber) {
     // if timestamp not in subgraph, fallback to onchain data
     timestampCreated = String(
-      (await parentProvider.getBlock(depositTx.blockNumber)).timestamp * 1000
+      normalizeTimestamp(
+        String((await parentProvider.getBlock(depositTx.blockNumber)).timestamp)
```

Comment:
Would we need `String(...)` here, since the date can now be `number | string | BigNumber`?

Reply:
good catch, updated